app/spiffworkflow/PropertiesPanel/parts/DataObjectArray.js

```diff
@@ -0,0 +1,142 @@
+import { useService } from 'bpmn-js-properties-panel';
+import { isTextFieldEntryEdited, TextFieldEntry } from '@bpmn-io/properties-panel';
+import { without } from 'min-dash';
+
+/**
+ * Provides a list of data objects, and allows you to add / remove data objects, and change their ids.
+ * @param props
+ * @constructor
+ */
+export function DataObjectArray(props) {
+  const moddle = props.moddle;
+  const element = props.element;
+  const process = props.element.businessObject;
```

Comment:
is "businessObject" their name and process "our" name?

Reply:
All visual elements (User Task icon, or a Data Object icon) can have an associated BusinessObject.  It made more sense to call this "process" because the business object of the selected element should be a BPMN Process.  So we are making that assumption.   I added a comment in the code to clarify.